gutenberg-ramp.php

```diff
@@ -129,16 +147,12 @@ function gutenberg_ramp_require_gutenberg() {
 
 	// perform any actions required before loading gutenberg
 	do_action( 'gutenberg_ramp_before_load_gutenberg' );
-	$gutenberg_include = apply_filters( 'gutenberg_ramp_gutenberg_load_path', WP_PLUGIN_DIR . '/gutenberg/gutenberg.php' );
 
-	if (    false === $gutenberg_include
-		||  0 !== validate_file( $gutenberg_include )
-		||  ! file_exists( $gutenberg_include )
-	) {
-		return false;
+	$validated_load_path = gutenberg_ramp_validated_gutenberg_load_path();
```

Comment:
Related to the earlier comment:

```suggestion
	$validated_load_path = gutenberg_ramp_get_validated_gutenberg_load_path();
```